l10n_it_payment_reason/migrations/14.0.1.0.1/pre-migrate.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Sergio Zanchetta <https://github.com/primes2h>
+# Copyright 2021 Marco Colombo <https://github.com/TheMule71>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.update_module_names(
+        env.cr,
+        [
+            ("l10n_it_causali_pagamento", "l10n_it_payment_reason"),
+        ],
+    )
+    openupgrade.rename_models(
+        env.cr,
+        [
+            ("causale.pagamento", "payment.reason"),
+        ],
+    )
```

Comment:
Penso che manchi [rename_tables](https://github.com/OCA/openupgradelib/blob/01cc9a1a8933376dcf6516b9a38030c6d0f16072/openupgradelib/openupgrade.py#L694) in base a quanto indicato [qui](https://github.com/OCA/openupgradelib/blob/01cc9a1a8933376dcf6516b9a38030c6d0f16072/openupgradelib/openupgrade.py#L767-L768).
L'ideale è testarlo così come ci siamo detti in standup:
- installare il modulo versione `14.0.1.0.0`
- aggiungere dei dati
- fare checkout sul commit in cui viene fatto il rename, versione `14.0.1.0.1`
- aggiornare il modulo